src/site/content/en/fast/incorporate-performance-budgets-into-your-build-tools/index.md

```diff
@@ -107,26 +136,24 @@ By default, **it tests gzipped asset sizes**. You can use the compression option
   </figcaption>
 </figure>
 
-[Lighthouse](https://developers.google.com/web/tools/lighthouse/) is an auditing tool that tests sites in a few key areas — performance, accessibility, best practices and how well your site performs as a progressive web application. You can run it in Chrome DevTools, from the command line, or as a Node module.
+[Lighthouse Bot](https://github.com/ebidel/lighthouse-ci) integrates with Travis CI and enforces budgets based on any of the five Lighthouse audit categories. It’s sometimes simpler to keep an eye on a single number than individual asset budgets and Lighthouse scores take many things into account.
 
 <figure class="w-figure">
-  <img class="screenshot" src="./lighthouse-scores.jpg" alt="Lighthouse scores 💯">
+  <img class="screenshot" src="./lighthouse-scores.png" alt="Lighthouse scores 💯">
```

Comment:
Could you grab a new screenshot of the Lighthouse report? Might as well change it while this article is being updated.

Reply:
I did that, you should be seeing this: [lighthouse-scores.png](https://github.com/GoogleChrome/web.dev/blob/perf-budgets/src/site/content/en/fast/incorporate-performance-budgets-into-your-build-tools/lighthouse-scores.png)

Not sure what went wrong?

Reply:
Weird. lighthouse-scores.png still looks like the old version to me.

This is what Lighthouse looks like for me locally:
<img width="895" alt="Screen Shot 2019-07-08 at 1 11 39 PM" src="https://user-images.githubusercontent.com/6076184/60829082-1adca900-a182-11e9-8a6b-314c83a9d72c.png">

Reply:
Oh you mean *that* new version. My screenshot was from the latest stable Chrome (75), I'll just grab one from the extension 👍